src/app/references/info/info.component.html

```diff
@@ -0,0 +1,5 @@
+<h3 class="mat-h3">Collection doesn't exist!</h3>
+
+<div mat-dialog-actions>
+  <button mat-button (click)="onNoClick()">Ok</button>
```

Comment:
onNoClick, but the text is Ok - ambiguous. Click handler might be just: close()

Reply:
Also need to change file info.component.ts
```suggestion
  <button mat-button (click)="close()">Ok</button>
```